.github/workflows/build.yml

```diff
@@ -1,10 +1,12 @@
+name: PR Workflow
+
 on: [push]
 jobs:
     tests:
       runs-on: ubuntu-latest
-      container: golang:1.15.5-alpine3.12
+      container: golang:1.16.5-alpine3.13
```

Comment:
Upgrade metro `go.mod` as well to 1.6
https://github.com/razorpay/metro/blob/d29e1cfc7a0b09bff5b9099bebe2f478f5e6f4c6/go.mod#L3